futures/Cargo.toml

```
@@ -56,3 +56,6 @@ read-initializer = ["futures-io/read-initializer", "futures-util/read-initialize
 
 [package.metadata.docs.rs]
 all-features = true
+
+[package.metadata.playground]
+features = ["std", "async-await", "compat", "io-compat", "executor", "threadpool"]
```

```suggestion
features = ["std", "async-await", "compat", "io-compat", "executor", "thread-pool"]
```

I renamed 'threadpool' feature to 'thread-pool' in #1937: https://github.com/rust-lang-nursery/futures-rs/pull/1937/files#diff-41c03002d828b5290c4f39adc8d88060R45